rhea/build/boards/altera/_de1_SoC.py

```diff
@@ -0,0 +1,229 @@
+#
+# Copyright (c) 2015 Christopher L. Felton
+#
+# 
+# Added Support for the Terasic De1-SoC Board
+# By Mike Kennedy (Va3TeC)
+#
+
+from __future__ import absolute_import
+
+from string import Template
+
+from rhea.build import FPGA
+from rhea.build.toolflow import Quartus
+
+
+class DE1SoC(FPGA):
+    vendor = 'altera'
+    family = 'Cyclone V'
+    device = '5CSEMA5F31C6'
+    speed = '6'
+    _name = 'de1_SoC'
+
+
+# As Per page 23 of the user Guide for DE1-SoC
+    default_clocks = {
+        'clock': dict(frequency=50e6, pins=('AF14',)) 
+        #'clock2': dict(frequency=50e6, pins=('AA16',))
```

Comment:
why are these commented out?  Is this what they are called in the documentation?  `clock2`, etc.